ui/lib/core/addon/components/string-list.js

```diff
@@ -63,7 +63,7 @@ export default class StringList extends Component {
 
   toVal() {
     const inputs = this.inputList.filter((x) => x.value).mapBy('value');
-    if (this.format === 'string') {
+    if (this.args.type === 'string') {
```

Comment:
I couldn't find `format` used as an arg anywhere so it seems like this was a random bug catch. Not sure why the test wasn't failing before :man_shrugging: